_ZadanieDomowe_8_PokerHands_Base/PokerHands/CardService.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace PokerHands
+{
+    public class CardService
```

Comment:
coś mi to podobnie wygląda do wersji kogoś innego, ale nie wnikam, kto zrzynał, i nie odejmuję.
Być może czytelniejsze by było nazwanie tego CardParser i przeniesienie dwóch ostatnich metod stąd do innej klasy lub ew. pozostawienie ich w Evaluatorze.

Reply:
tak czy inaczej plus za wydzielenie